src/js/components/Calendar/Calendar.js

```diff
@@ -231,7 +231,15 @@ class Calendar extends Component {
     return (
       <Box direction="row" justify="between" align="center">
         <Box flex pad={{ horizontal: headingPadMap[size] || 'small' }}>
-          <Heading level={size === 'small' ? 4 : 3} size={size} margin="none">
+          <Heading
+            level={
+              size === 'small'
+                ? theme.calendar.heading && theme.calendar.heading.level
+                : 3
```

Comment:
This might be a problem if the user had defined the `theme.calendar.heading.level: 2`, and we are setting the level to be level of 3 when `size` is small, which wouldn't be proportional.
I'm thinking this should be `theme.calendar.heading.level - 1` instead of hardcoded '3'.
@ericsoderberghp what do you think about this approach?